docs/learn/concepts.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: '📑 Major Concepts'
```

Comment:
Maybe we could also use the term "Glossary" as well? It seems to fit the purpose of this page.

```suggestion
sidebar_label: '📑 Concepts & Glossary'
```

or 

```suggestion
sidebar_label: '📑 Major Concepts& Glossary'
```